src/test/Vault.t.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.16;
 import {TestPlus} from "./TestPlus.sol";
 import {stdStorage, StdStorage} from "forge-std/Test.sol";
 
-import {Vault} from "../ethereum/Vault.sol";
+import {Vault} from "../both/Vault.sol";
```

Comment:
whys it called `both`--maybe there is a better clearer naming?

Reply:
Created this bug
https://linear.app/alpine-camp/issue/ALP-3441/rename-the-both-folder-to-something-more-meaningful